src/ER_V/acl/acl_private.h

```diff
@@ -0,0 +1,64 @@
+#include "acl/acl.h"
+#define CMD_SIZE 32
+
+#define VAR_POS "DELTA" /** variable position moniker for ACL commands */
+
+/** Conversion factors for Encoder Count */
+#define BASE_CONVERSION_FACTOR 42.5666f
+#define SHOULDER_CONVERSION_FACTOR 33.2121f
+#define WRIST_CONVERSION_FACTOR 8.3555f
+
+#define ACL_CMD_COUNT 10
+
+/** ACL Command Formats */
+#define SHIFT "SHIFT %s BY %u %d\r"
+#define HERE "HERE %s"
+#define MOVE "MOVE %s\r"
+
+/** ACL Command Types */
+typedef enum _acl_command_type
+{
+    CMD_HOME,
```

Comment:
This all looks good; It'd probably be a good idea to breifly describe the ACL command itself as a comment next to it